ppr-ui/.eslintrc.js

```diff
@@ -7,8 +7,13 @@ module.exports = {
     'plugin:vue/essential',
     '@vue/standard',
     '@vue/typescript'
+    // ToDo: enable and fix Lint issues when capacity.
+    // 'plugin:vue/base',
+    // 'plugin:vuetify/base',
+    // 'plugin:vuetify/recommended'
   ],
   rules: {
+    'vue/multi-word-component-names': 'off',
```

Comment:
Feels like a weird rule to me, so i turned it off. Anyone else?
ie `Folio` wouldn't be accepted but `FolioNumber` or `Folio-Number` would be. 
I'm not sure the basis but there is probably a reason, that being said though, didn't want to rename all the single name components in the application :P

Reply:
Disabling makes sense to me.